tests/config/ios/INTERFACE_DOWN/time_synchronized/syslog.msg

```diff
@@ -0,0 +1 @@
+<189>117: NetAuto_CSRv-03: May  9 12:49:27.098: %LINK-5-CHANGED: Interface GigabitEthernet2, changed state to administratively down
```

Comment:
Along side this file you should add a file named `yang.json` with the following contents:

```
{
  "yang_message": {
    "interfaces": {
      "interface": {
        "GigabitEthernet2": {
          "state": {
            "admin_status": "DOWN"
          }
        }
      }
    }
  },
  "message_details": {
    "severity": 5,
    "facility": 23,
    "messageId": "117",
    "pri": "189",
    "host": "NetAuto_CSRv-03",
    "tag": "LINK-5-CHANGED",
    "time": "12:49:27",
    "date": "May 9",
    "message": "Interface GigabitEthernet2, changed state to administratively down",
    "milliseconds": "098"
  },
  "facility": 23,
  "ip": "127.0.0.1",
  "error": "INTERFACE_DOWN",
  "host": "NetAuto_CSRv-03",
  "yang_model": "openconfig-interfaces",
  "timestamp": 1525870167,
  "os": "ios",
  "severity": 5
}
```

You can find the expected output in the failed travis job:

https://travis-ci.org/napalm-automation/napalm-logs/jobs/377001016

You can also find the expected output by running locally, sending the syslog message to the listener, you will then see the dict output in the client. Then you will need to convert this to `json`, you can use https://jsoneditoronline.org/